### ISSUE TITLE
Consider adding google/dagger

### ISSUE THREAD

**Issue by perezd** [google/bazel-common#38]
Could be a worthwhile addition to this since it requires some annotation processor setup as well much like Auto.

**Comment by ronshapiro** [comment]
You're right that we should do this. I haven't figured out exactly how we should be interfacing with libraries that use bazel-common yet, so I'm hesitant to rush to something too quickly. But if you use bazel-common and then add a workspace dep on Dagger, everything should work

**Comment by perezd** [comment]
Oh I suspect it will work, I just want bazel-common to do it for me :)

**Comment by perezd** [comment]
I think its probably best to just depend on the jars from maven, seems like Bazel-based integration is broken pretty badly at the moment.

**Comment by ronshapiro** [comment]
I don't think that's a reason to rush this unfortunately. I don't think it's something we want long term.

But we can fix any of those issues (thanks for reporting them)! so that we can get things up to speed.
